content/themes/glimmer/partials/sidebar.hbs

```diff
@@ -1,59 +1,60 @@
-                        document.write('{{> "widgets/social-widget"}}');
-                    } else if ( Sidebar_Widget_Order[key] === "Facebook_Page_Widget" && Facebook_Page_Widget['show'] === true  ) {
-                        document.write('{{> "widgets/facebook-widget"}}');
-                    } else if ( Sidebar_Widget_Order[key] === "Googleplus_Page_Widget" && Googleplus_Page_Widget['show'] === true  ) {
-                        document.write('{{> "widgets/google-plus-widget"}}');
-                    } else if ( Sidebar_Widget_Order[key] === "Twitter_Widget" && Twitter_Widget['show'] === true  ) {
-                        document.write('{{> "widgets/twitter-widget"}}');
-                    } else if ( Sidebar_Widget_Order[key] === "Flickr_Widget" && Flickr_Widget['show'] === true  ) {
-                        document.write('{{> "widgets/flickr-widget"}}');
-                    } else if ( Sidebar_Widget_Order[key] === "Mailchimp_Widget" && Mailchimp_Widget['show'] === true && Cookies.get('#subscribe-modal') != 'subscribed' ) {
-                        document.write('{{> "widgets/newsletter-widget"}}');
-                    } else if ( Sidebar_Widget_Order[key] === "Latest_Posts_Widget" && Latest_Posts_Widget['show'] === true  ) {
-                        document.write('{{> "widgets/latest-posts-widget"}}');
-                    } else if ( Sidebar_Widget_Order[key] === "Tags_Widget" && Tags_Widget['show'] === true  ) {
-                        document.write('{{> "widgets/tags-widget"}}');
-                    } 
-                } 
-                
-            </script> 
-
-	</div> <!-- #secondary -->	
+{{#has tag="sidebar-left, sidebar-right"}}
+    {{#has tag="sidebar-left"}}
+        <div class="col-md-4 col-md-pull-8" >
+    {{/has}}
+    {{#has tag="sidebar-right"}}
+        <div class="col-md-4">
+    {{/has}}
+{{else}}
+
+    {{#is "post, page"}}
+        {{#is "post"}}
+        <div class="col-md-4" id="post-sidebar-layout">
+        {{/is}}
+        {{#is "page"}}
+        <div class="col-md-4" id="page-sidebar-layout">
+        {{/is}}
+    {{else}}
+        <div class="col-md-4" id="sidebar-layout">
+    {{/is}}
+
+{{/has}}
+
+	<!-- Sidebar -->
+	<div id="secondary" class="widget-area">
+
+            <script>
+
+                // Showing Widgets by order
+                for ( var key in Sidebar_Widget_Order ) {
+
+                    if ( Sidebar_Widget_Order[key] === "About_Me_Widget" && About_Me_Widget['show'] === true  ) {
+                        document.write('{{> "widgets/about-widget"}}');
+                    } else if ( Sidebar_Widget_Order[key] === "Ads_Widget" && Ads_Widget['show'] === true  ) {
+                        document.write('{{> "widgets/ads-widget"}}');
+                        $('.ads-widget a')[0].href = Ads_Widget['url'];
+                        $('.widget_ads .widget-title-area').remove();
```

Comment:
Unfortunately have to do some wrangling with JS to manipulate some of the handlebar generated code or else we get JSErrors when writing the handlebar widget to the DOM. and the page bombs out (modules don't load).